### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by Didgeridrew** [kdw2060/hassio-addons#17]
When starting the add-on I receive the following error in the logs:

```
s6-overlay-suexec: fatal: can only run as pid 1
```

https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images/

**Comment by kdw2060** [comment]
Hi, thanks for pointing me to that blogpost. I guess the change does not affect all Home Assistant flavors, cause on my supervised install I get no error like that (running the latest Home Assistant version).
I have however updated the addon configuration file as instructed in the blogpost and pushed the update to github. Can you install the update and verify this fixes the issue for you?

**Comment by kdw2060** [comment]
I reverted the change since another issue has popped up. Waiting on an update from the HA dev community to get that and this issue fixed in one go.

**Comment by kdw2060** [comment]
see #18 > has nothing to do with your server. I hope to get feedback from the HA dev community soon

**Comment by kdw2060** [comment]
Found a solution, should work again when you update to 0.202

**Comment by OneOfThePetes** [comment]
I'm getting s6-overlay-suexec: fatal: can only run as pid 1

**Comment by kdw2060** [comment]
Did a new install and cannot reproduce this issue. For me it was fixed since v 0.202
I'm sadly also no specialist in docker who can help you deduce what's the specific issue you might have.
